sale_order_line_sequence/models/account_move_line.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2023 Forgeflow S.L.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+
+
+class AccountMoveLine(models.Model):
+    _inherit = "account.move.line"
+
+    related_so_sequence = fields.Char(
+        string="SO Line Number",
+        compute="_compute_related_so_sequence",
+    )
+
+    @api.depends("move_id.invoice_line_ids")
+    def _compute_related_so_sequence(self):
+        for rec in self:
+            if len(rec.mapped("move_id.line_ids.sale_line_ids.order_id")) > 1:
```

Comment:
Given that you are touching this line in the migration commit, better go all the way with 
```suggestion
            if len(rec.move_id.line_ids.sale_line_ids.order_id) > 1:
```